apps/aeoracle/src/aeo_query_tx.erl

```diff
@@ -80,17 +93,19 @@ response_ttl(#oracle_query_tx{response_ttl = ResponseTTL}) ->
     ResponseTTL.
 
 -spec new(map()) -> {ok, aetx:tx()}.
-new(#{sender        := SenderPubKey,
+new(#{sender        := SenderAccount,
       nonce         := Nonce,
-      oracle        := OraclePubKey,
+      oracle        := Oracle,
       query         := Query,
       query_fee     := QueryFee,
       query_ttl     := QueryTTL,
       response_ttl  := ResponseTTL,
       fee           := Fee} = Args) ->
-    Tx = #oracle_query_tx{sender        = aec_id:create(account, SenderPubKey),
+    account = aec_id:specialize_type(SenderAccount),
+    oracle  = aec_id:specialize_type(Oracle), %% TODO: Should also be 'name'
```

Comment:
When enabling oracle query target to be name by specifying its hash, Swagger param [`oracle_pubkey`](https://github.com/aeternity/epoch/blob/5a72c277eb0f8619d3c7f8af5c94e63263db10df/config/swagger.yaml#L2558) shall probably be renamed (as a name hash is not a pubkey).